include/sys/stat.h

```diff
@@ -62,6 +62,10 @@
 #define S_IRUSR     (1 << 8)
 #define S_IRWXU     (7 << 6)
 
+#define S_IREAD     S_IRUSR
+#define S_IWRITE    S_IWUSR
+#define S_IEXEC     S_IXUSR
```

Comment:
```suggestion
#define S_IREAD     S_IRUSR     /* Obsolete synonym provided for BSD compatibility. */
#define S_IWRITE    S_IWUSR    /* Obsolete synonym provided for BSD compatibility. */
#define S_IEXEC     S_IXUSR      /* Obsolete synonym provided for BSD compatibility. */
```
Should we add this disclaimer?

Reply:
Yes, looks reasonable.

Reply:
Done.